renderer/components/Pay/PayPanelFooter.js

```diff
@@ -49,6 +50,22 @@ const getNextButtonText = (formState, props) => {
   return nextButtonText
 }
 
+const getLiquidityWarningMessage = (hasEnoughFunds, capacity, unit) => {
```

Comment:
I find the logic of this method a little confusing...

If the user doesn't have enough funds it returns a "not enough funds error". Otherwise it returns an "one time send capacity" error.

The reason for the first error is obvious since its handled by `if (!hasEnoughFunds) {` but it's not obvious why it defaults to the "one time send capacity" error otherwise.

I'd suggest breaking this out into 2 separate methods and calling slightly more explicitly:

```js
if (!hasEnoughFunds) {
  getNotEnoughFundsError()
} else if (isAboveMax) {
  getAboveMaxSendCapacityError(capacity, unit))
}
```

Reply:
i've redone that entire part from scratch